networkconfig/config.go

```diff
@@ -77,6 +77,5 @@ func (n NetworkConfig) GetGenesisTime() time.Time {
 }
 
 func (n NetworkConfig) AlanFork() bool {
-	// TODO: implement
-	return true
+	return n.Beacon.EstimatedCurrentEpoch() >= n.AlanForkEpoch
 }
```

Comment:
we should have this or the other not both.

Reply:
Agree. I was not sure does it matter to provide slot as param or not for `AlanForked(slot)`